content/2019-09-23_PyConCZ_wrap_up_summary.md

```diff
@@ -0,0 +1,215 @@
+
+### Financial Aid
+
+Within the constraints of our budget, we tried to cover travel, accommodation and ticket prices to those who wouldn’t otherwise be able to afford it.
+
+Speakers and workshop leaders were offered free tickets and we spent over 110 000 CZK to help finance their travel and accommodation.
+
+We also gave out 11 regular free tickets (mostly covered by sponsors) and spent 5600 CZK to help two attendees with costs of travel and accommodation.
+
+We sadly had to decline aid to attendees asking for larger amounts (over 400 EUR/person) as our budget was limited.
+
+*Next year, we plan to set an upper limit on grants to focus on providing smaller grants to more people. We will try to reach out to underrepresented groups in Czech Republic better.*
+
+### Accessibility
+
+Both main and workshop venues were wheelchair-accessible.
+
+*Next year we aim to focus on promoting wheelchair-accessibility to be more attractive to people with disabilities.*
+
+### Languages
+
+Except one talk in Czech, we kept the event international with English as the main language. 
+
+While almost 70 % of attendees understood Czech (this includes Slovaks) we managed to attract 30 % of people from abroad who might not be able to attend otherwise.
+
+*We plan to keep English as the main language but we’re considering trying to attract more beginners by having an extra Czech-language programme for beginners. This should not limit the enjoyment of the event by foreign attendees, as they tend to be more advanced Pythonistas.*
+
+### Gender
+
+We had 32 talks of which 25 (78 %) were by men and 7 (22 %) by women. At workshops, there were 14 tutors in total, of which 12 (86 %) were men and 2 (14 %) were women. In total we had 41 people talking or leading a workshop of which 7 (17 %) were women.
+
+Regarding all attendees, there were 77 % men and 23 % women, a bit less than in 2018 but more than PyCon CZ 2017.
+
+*For 2020 we would like to promote PyCon CZ as more interesting for women speakers.*
+
+### Code of Conduct Report
+
+The Code of Conduct was kept the same as in the previous years, based on the Conference anti-harassment policy from the Geek Feminism Wiki page and edited to be as specific to our event as possible.
+
+Agreeing to the Code was mandatory when buying the ticket.
+
+Two contact persons (male and female) were assigned.
+
+Attendees were reminded of the Code in the opening speech of each day and on the badge they were wearing.
+
+One incident regarding tasteless commentary about a different programming language during a presentation was reported this year. The reporter was assured PyCon CZ doesn’t identify with the message. Next day MC emphasized we all should treat each other nice. The staff followed up with the presenter about the inappropriate remark. Presenter expressed an apology on their social networks after the conference.
```

Comment:
Co je MC?

Reply:
Master of Ceremonies

Reply:
Souhlas, to by asi chtelo rozepsat.

Reply:
Aha. Jsem si říkal, že magnetofonová kazeta to asi nebude, Master of Ceremonies je taky blbost... 😄

Reply:
Taky jsem slysel tomu rikat "microphone controller", ale to nema to spravne dekorum :D

Reply:
Můžete přímo zde navrhnout změnu.

Reply:
Změnit třeba na `moderator` nebo nechat tak?

Reply:
Vyřešeno Petrem.